dist/classes/index.js

```diff
@@ -1,6 +1,6 @@
-"use strict";
-function __export(m) {
-    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
-}
-Object.defineProperty(exports, "__esModule", { value: true });
-__export(require("./Session"));
+"use strict";
+function __export(m) {
+    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
+}
+Object.defineProperty(exports, "__esModule", { value: true });
+__export(require("./Session"));
```

Comment:
@chrispanag You have committed a bunch of files without changes! Only whitespace/LE. Please soft reset your branch and remove them and push again.

Reply:
Indeed. 
tsc is to blame. Maybe update the `.gitignore` to only take `src` into account (adding `dist` to `.gitignore`)

Reply:
Sure but there's no need for this change now. Please revert these changes and check my other comments and I'd be happy to merge!